packages/atti-components/src/components/Toggle/Readme.mdx

```diff
@@ -0,0 +1,58 @@
+---
+path: '/toggle'
+name: 'Toggle'
+title: 'Toggle'
+---
+
+Toggle offers the user the possibility to choose or not item.
+
+## Example
+
+```js
+<div
+  style={{
+    paddingBottom: '30px',
+    display: 'flex',
+    justifyContent: 'space-between',
+    margin: '0 48px 0 -10px',
+  }}
+>
+  <div>
+    <Toggle id="toggleTest1" name="toggleTestGroup" value="ImChecked" />
+  </div>
+  <div>
+    <Toggle
+      id="toggleTest2"
+      name="toggleTestGroup"
+      value="ImChecked"
+      variant="neutral"
```

Comment:
I'm not really fan of this variant + of the basic component. According to me it's really complicated to know if it's on or off.

Reply:
I suggest this new solution. I've changed the terms for clarity. Is that all right?

Reply:
Sorry, maybe my comment was not really clear. It's more that visually it's too complicated to know if it's on or off. For the last version, it's clear as we are switching from red to green.
For the first 2, it's the same color for on and off states.
So maybe you can just change it as it was before (if I'm right, "color" prop wasn't there) and just "play" a bit with the color ;)

Reply:
I've modified it, you can see the contrast better when the toggle is on and when it's off. :)